README.md

```diff
@@ -87,12 +88,13 @@ A curated list of awesome open source workflow engines
 * [Camunda](https://camunda.com) [![Stars](https://img.shields.io/github/stars/camunda/camunda-bpm-platform.svg)](https://github.com/camunda/camunda-bpm-platform) - BPMN-based workflow engine that can be embedded as java library (e.g. Spring Boot) or used standalone, including a graphical modeler and operations tooling.
 * [Captain](https://github.com/LiveRamp/captain) [![Stars](https://img.shields.io/github/stars/LiveRamp/captain.svg)](https://github.com/LiveRamp/captain) - Distributed, light-weight java workflow engine for a microservice architecture.
 * [CoreWF](https://github.com/UiPath/corewf) [![Stars](https://img.shields.io/github/stars/UiPath/corewf.svg)](https://github.com/UiPath/corewf) - WF runtime ported to work on .NET Core
+* [ctrl+flow](https://github.com/ctrlflowjs/ctrlflow) [![Stars](https://img.shields.io/github/stars/ctrlflowjs/ctrlflow.svg)](https://github.com/ctrlflowjs/ctrlflow) - Fullstack JS framework to embed visual programming interfaces in web apps, with durable background processing. Delivers an intuitive UX that turns non-technical users into collaborators and reduces load on dev teams.
 * [Dagger](https://github.com/wayfair-incubator/dagger) - Dagger is a distributed, horizointally scalable, durable, and highly available orchestration engine in python based on [Faust-Streaming](https://github.com/faust-streaming/faust) for running millions of long running tasks with direct integration with Kafka
 * [Django River](https://github.com/javrasya/django-river) [![Stars](https://img.shields.io/github/stars/javrasya/django-river.svg)](https://github.com/javrasya/django-river) - Django workflow library that supports on the fly changes for states, transitions, and authorizations
 * [Kogito](https://kogito.kie.org/) [![Stars]( https://img.shields.io/github/stars/kiegroup/kogito-runtimes.svg )](https://github.com/kiegroup/kogito-runtimes) - Cloud-native business automation technology. Embeddable to produce JVM or GraalVM artifacts or interacted through Kafka & REST APIs
 * [Luigi](https://github.com/spotify/luigi) [![Stars](https://img.shields.io/github/stars/spotify/luigi.svg)](https://github.com//spotify/luigi) - Python module that helps you build complex pipelines of batch jobs.
 * [SciPipe](https://github.com/scipipe/scipipe) [![Stars](https://img.shields.io/github/stars/scipipe/scipipe.svg)](https://github.com/scipipe/scipipe) - A Go library for writing pipelines of Bash commands or Go-code using the dataflow / flow-based programming paradigm.
-* [SpiffWorkflow](https://spiffworkflow.readthedocs.io/en/latest/) [![Stars](https://img.shields.io/github/stars/sartography/SpiffWorkflow.svg)](https://github.com/sartography/SpiffWorkflow) - SpiffWorkflow - a BPMN 2.0 workflow engine implemented in pure Python.  
```

Comment:
Forgive the confusing additional diffs, my IDE automatically removes trailing whitespace on lines.